plugins/org.eclipse.osee.disposition.rest.model/src/org/eclipse/osee/disposition/model/DispoAnnotationData.java

```diff
@@ -241,18 +267,58 @@ public void setResolution(String resolution) {
    }
 
    public void setLastResolutionType(String lastResolutionType) {
+      if (lastResolutionType == null) {
+         return;
+      }
+      this.lastResolutionType = lastResolutionType;
+   }
+
+   public void setFilledLastResolutionType(String lastResolutionType) {
+      if (lastResolutionType == null || lastResolutionType.isEmpty() || lastResolutionType.isBlank()) {
+         return;
+      }
       this.lastResolutionType = lastResolutionType;
    }
 
    public void setLastResolution(String lastResolution) {
+      if (lastResolution == null) {
+         return;
+      }
+      this.lastResolution = lastResolution;
+   }
+
+   public void setFilledLastResolution(String lastResolution) {
+      if (lastResolution == null || lastResolution.isEmpty() || lastResolution.isBlank()) {
+         return;
+      }
       this.lastResolution = lastResolution;
    }
 
    public void setLastManualResolutionType(String lastManualResolutionType) {
+      if (lastManualResolutionType == null) {
+         return;
+      }
+      this.lastManualResolutionType = lastManualResolutionType;
+   }
+
+   public void setFilledLastManualResolutionType(String lastManualResolutionType) {
+      if (lastManualResolutionType == null || lastManualResolutionType.isEmpty() || lastManualResolutionType.isBlank()) {
+         return;
+      }
       this.lastManualResolutionType = lastManualResolutionType;
    }
 
    public void setLastManualResolution(String lastManualResolution) {
+      if (lastManualResolution == null) {
```

Comment:
can use ternary to make it cleaner if you want. this.lastManualResolution = !lastManualResolution.equals(null) ? lastManualResolution : "";